core/controllers/learner_dashboard_test.py

```diff
@@ -301,10 +299,35 @@ def test_get_message_summaries(self):
         messages_summary = response_dict['message_summary_list']
 
         # Check the summary of the second message.
-        second_message = messages_summary[1]
+        second_message = messages_summary[-1]
         self.assertDictContainsSubset({
             'text': 'Message 1',
             'author_username': 'editor'
         }, second_message)
 
         self.logout()
+
+    def test_feedback_without_author_settings(self):
```

Comment:
Can maybe pick a better name here: 
```suggestion
    def test_anonymous_feedback_is_recorded_correctly(self):
```

Reply:
Done